R/codelist.R

```diff
@@ -50,12 +50,18 @@
 #'   \item p4.name: Polity IV country name
 #'   \item region: 7 Regions as defined in the World Bank Development Indicators
 #'   \item region23: 23 Regions as used to be in the World Bank Development Indicators (legacy)
+#'   \item un.intermediateregion.name: United Nations intermediate region name
+#'   \item un.intermediateregion.name: United Nations intermediate region code
 #'   \item un.name.ar: United Nations Arabic country name
 #'   \item un.name.en: United Nations English country name
 #'   \item un.name.es: United Nations Spanish country name
 #'   \item un.name.fr: United Nations French country name
 #'   \item un.name.ru: United Nations Russian country name
 #'   \item un.name.zh: United Nations Chinese country name
+#'   \item un.region.name: United Nations region name
+#'   \item un.region.name: United Nations region code
+#'   \item un.subregion.name: United Nations sub-region name
+#'   \item un.subregion.name: United Nations sub-region code
```

Comment:
probably should be `\item un.subregion.code: United Nations sub-region code`

Reply:
Oops